src/pages/payment-methods/_payment-data.js

```diff
@@ -359,7 +359,12 @@ const payment_data = [
             // },
             {
                 method: <StyledIcon src={PaysafeCard} alt="PaysafeCard" />,
-                currencies: 'USD AUD GBP EUR',
+                currencies: (
+                    <Localize
+                        translate_text="USD AUD <0></0>GBP EUR"
+                        components={[<br key={0} />]}
+                    />
+                ),
```

Comment:
You should be able to introduce the break without having to use the `Localize` component right? Since the rest of the currencies aren't translated.

Reply:
Amended in the latest commit.